src/Adapters/Auth/gpgames.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+// Helper functions for accessing the google API.
+var https = require('https');
+var Parse = require('parse/node').Parse;
+var request = require('request');
+
+// Returns a promise that fulfills if this user id is valid.
+function validateAuthData(authData, authOptions)
+{
```

Comment:
again, keep the curly braces up on the same line. It's inconsistent throughout this file as well if you can amend that.